projects/dragonzil/project.md

```diff
@@ -1,10 +1,10 @@
 ---
 name: DragonZIL
 description: A game that allows you to buy digital collectible "dragons" with ZIL, which mate, fight and mutate.
-categories: gaming,dapp
+categories: gaming,dapp,gambling,NFT,earn
```

Comment:
1. please remove categories `gambling` and `earn`. 
2. lowercase category `NFT`